src/components/BaseStats.js

```diff
@@ -12,79 +12,80 @@ const BaseStats = ({ stats, types }) => {
-            labels: [
-              "HP",
-              "Attack",
-              "Defense",
-              "Sp. Attack",
-              "Sp. Defense",
-              "Speed",
-            ],
-            datasets: [
-              {
-                data: [...baseStatsArray],
-                backgroundColor: [getTypeColors[typeColor]],
-                hoverBackgroundColor: [getTypeColors[typeColor]],
-                borderRadius: 50,
-                borderSkipped: false,
-                borderWidth: 1,
-                barThickness: 16,
+        {/* show bar graph or spinner by checking for baseStatsArray */}
+        {baseStatsArray ? (
+          <Bar
+            placeholder={"hi"}
+            data={{
+              labels: [
+                "HP",
+                "Attack",
+                "Defense",
+                "Sp. Attack",
+                "Sp. Defense",
+                "Speed",
+              ],
+              datasets: [
+                {
+                  data: [...baseStatsArray],
+                  backgroundColor: [getTypeColors[typeColor]],
+                  hoverBackgroundColor: [getTypeColors[typeColor]],
+                  borderRadius: 50,
+                  borderSkipped: false,
+                  borderWidth: 1,
+                  barThickness: 16,
+                },
+              ],
+            }}
+            height={200}
+            width={600}
+            options={{
+              layout: {
+                padding: 10,
               },
-            ],
-          }}
-          height={200}
-          width={600}
-          options={{
-            layout: {
-              padding: 10,
-            },
-            indexAxis: "y",
-            elements: {
-              bar: {
-                borderWidth: 2,
+              indexAxis: "y",
+              elements: {
+                bar: {
+                  borderWidth: 2,
+                },
               },
-            },
-            responsive: true,
-            scales: {
-              x: {
-                grid: {
-                  display: false,
-                  drawBorder: false,
+              responsive: true,
+              scales: {
+                x: {
+                  grid: {
+                    display: false,
+                    drawBorder: false,
+                  },
+                },
+                y: {
+                  grid: {
+                    display: false,
+                    drawBorder: false,
+                  },
                 },
               },
-              y: {
-                grid: {
+              maintainAspectRatio: false,
+              plugins: {
+                legend: {
                   display: false,
-                  drawBorder: false,
+                },
+                title: {
+                  display: false,
+                  text: "",
+                },
+                tooltip: {
+                  enabled: true,
+                  backgroundColor: "rgb(0,0,0,0.6",
```

Comment:
Might not be a big deal, but it looks like you're missing an ending parenthesis?